src/app/credExplorer/PagerankTable.js

```diff
@@ -92,14 +92,12 @@ export class PagerankTable extends React.PureComponent<
   }
 
   render() {
-    if (this.props.adapters == null) {
-      return <p>You must load a graph before seeing PageRank analysis.</p>;
-    }
-    if (this.props.pnd == null) {
-      return <p>Please run PageRank to see analysis.</p>;
-    }
-    if (this.props.maxEntriesPerList == null) {
-      throw new Error("maxEntriesPerList not set");
+    if (
```

Comment:
Do we really need this check?

Reply:
I thought about this as well. I don't like having a habit of unnecessarily checking invariants that the type system should maintain. However, due to https://github.com/facebook/flow/issues/6400 (and similar) I'm not sure I trust flow to actually maintain its checks.

That class of bug does seem pretty rare, and I don't think I want to establish a consistent policy of always checking required props on React component construction, so I'm happy to take the check out.